### PR TITLE
this has some tests that show issues with how we do this conversion

### PR DESCRIPTION
**Pull request by kris-watts-gravwell**
Basically because of the "clean" phase of the map fixup we introduce some ambiguity and failures.

I think we need to remove the clean phase and just specify the headers in the exact form that corelight throws them, we don't emit the headers anywhere, so its OK to keep them as is.

Test included.

example logs here: https://docs.zeek.org/en/master/logs/index.html

### CODE REVIEW

File: ingest/processors/corelight.go [hunk 1]
```diff
@@ -162,7 +154,10 @@ func (c *Corelight) process(mp map[string]interface{}, og []byte) (tag string, t
 	} else if line, ok = emitLine(ts, headers, mp); !ok {
 		tag = defaultTag
 		line = og
+	} else {
+		line = bytes.TrimRight(line, "-\t\n")
```
Comment (john-floren-gravwell) [inline ingest/processors/corelight.go:158]: Won't this trim off the last - column? Surely we want to keep that, if only for consistency's sake with files ingested natively as TSV

File: ingest/processors/corelight.go [hunk 2]
```diff
@@ -191,7 +186,7 @@ func (c *Corelight) getTagTs(mp map[string]interface{}) (tag string, ts time.Tim
 func emitLine(ts time.Time, headers []string, mp map[string]interface{}) (line []byte, ok bool) {
 	bb := bytes.NewBuffer(nil)
 	var f64 float64
-	fmt.Fprintf(bb, "%.3f", float64(ts.UnixNano())/1000000000.0)
+	fmt.Fprintf(bb, "%.5f", float64(ts.UnixNano())/1000000000.0)
```
Comment (john-floren-gravwell) [inline ingest/processors/corelight.go:189]: Looking at actual zeek logs, they appear to use 6 digits of precision:

```
1663714154.384267	CXGv4RcQXWSFXRzGd	192.168.0.77
```

File: ingest/processors/corelight.go (hunk 1 above)
Comment (kris-watts-gravwell) [reply ingest/processors/corelight.go]: fixed.

File: ingest/processors/corelight.go (hunk 2 above)
Comment (kris-watts-gravwell) [reply ingest/processors/corelight.go]: fixed

**Review by john-floren-gravwell — APPROVED** [review]

**Pull request merged**